spec/views/my_facilities/drug_stocks/drug_stocks.html.erb_spec.rb

```diff
@@ -0,0 +1,123 @@
+require "rails_helper"
+
+RSpec.describe "my_facilities/drug_stocks/drug_stocks.html.erb", type: :view do
+  let(:region) { instance_double("Region", id: 123) }
+  let(:facility) do
+    instance_double(
+      "Facility",
+      id: 1,
+      name: "Facility 1",
+      localized_facility_size: "Small",
+      region: region
+    )
+  end
+  let(:district1) { instance_double("FacilityGroup", id: 1, name: "District 1", slug: "district-1", state: "State 1") }
+  let(:district2) { instance_double("FacilityGroup", id: 2, name: "District 2", slug: "district-2", state: "State 2") }
+  let(:current_admin) { instance_double("Admin") }
+
+  helper do
+    def accessible_region?(*)
+      true
+    end
```

Comment:
I don't think we need this method. It's not used in the entire PR. Please check for the other helper methods defined here as well